modules/thisbook.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable */
+class thisBook {
+  constructor() {
+    if (localStorage.getItem('Added books') === null) {
+      localStorage.setItem('Added books', JSON.stringify([]));
+    }
+  }
+
+  getStoredBooks() {
+    return JSON.parse(localStorage.getItem('Added books'));
+  }
+
+  updateStoredBooks(books) {
+    localStorage.setItem('Added books', JSON.stringify(books));
+  }
+  addNewBook(bookTitle, bookAuthor) {
+    const storedBooks = this.getStoredBooks();
+    const newBook = {
+      title: bookTitle,
+      author: bookAuthor,
+    };
+    storedBooks.push(newBook);
+    this.updateStoredBooks(storedBooks);
+    this.displayBooks();
+  }
+
+  removeBook(i) {
+    const storedBooks = this.getStoredBooks();
+    storedBooks.splice(i, 1);
+    this.updateStoredBooks(storedBooks);
+    this.displayBooks();
+  }
+
+  createBookListHTML(books) {
+    const bookListHTML = books
+      .map((book) => {
+        return `
+        <div class= "booklist">
+        <p>"${book.title}" by "${book.author}"</p>
+        <button onClick="thisBook.removeBook(${books.indexOf(book)})">Remove</button>
+        </div>
+        `;
+      })
+      .join('');
+    return bookListHTML;
+  }
```

Comment:
Good job separating your code into `Modules` as expected in the requirements. 📁 However another requirement of this milestone is to refactor all your JavaScript functions and make them ES6 functions. 

- [ ] Please check and use arrow functions in all the functions as it is the recommended approach for modern development. 🚀